ci/00_setupgems.sh

```diff
@@ -1,10 +1,13 @@
 #!/bin/sh -xe
 
-gem install bundler --no-ri --no-rdoc
+# Show version to help debugging (or fail fast)
+yq --version
+ruby -v
+bundle -v
 
+# Install Unit Test Dependencies
 mkdir -p vendor/gems
+bundle install --without development plugins --path=vendor/gems
```

Comment:
```suggestion
bundle install --verbose --without development plugins --path=vendor/gems
```
Not sure if needed anymore

Reply:
The verbose mode for `bundle install` does not feel really useful. It happens when there is an issue with one of the dependencies.
But it adds a time penalty (twice the time on my local macOS :'( ) + it is not set on the actual Jenkinsfile on the principal branch, so I've removed it.